### PR TITLE
Reservation calendar

### PR DESCRIPTION
**Pull request by StephenDMay**
new branch, added functionality to number of child/infant buttons

### CODE REVIEW

File: findmyparadise/src/main/java/com/tp/wrc/findmyparadise/models/Host.java [hunk 1]
```diff
@@ -46,18 +45,10 @@
     @Column(name = "image_src", nullable = false)
     private String imageSrc;
 
-    @OneToMany(mappedBy = "host")
```
Comment (kpugh97) [inline findmyparadise/src/main/java/com/tp/wrc/findmyparadise/models/Host.java:49]: why did you remove listings from host?

**Review by tamera-brown — APPROVED** [review]
